NanoVNASaver/SweepWorker.py

```diff
@@ -56,35 +56,35 @@ class WorkerSignals(QtCore.QObject):
 
 class Sweep():
     def __init__(self, start: int = 3600000, end: int = 30000000,
-                 points: int = 101, sweeps: int = 1):
+                 points: int = 101, segements: int = 1):
```

Comment:
this has an extra 'e' in the middle

Reply:
Ok, fixed it ... luckily mistyping was also in assignment :+1: